lib/plugins/Label_H1_FPN.test.ts

```diff
@@ -18,22 +18,28 @@ test('decodes Label H1 Preamble FPN landing', () => {
   console.log(JSON.stringify(decodeResult, null, 2));
 
   expect(decodeResult.decoded).toBe(true);
-  expect(decodeResult.decoder.decodeLevel).toBe('partial');
+  expect(decodeResult.decoder.decodeLevel).toBe('full');
   expect(decodeResult.decoder.name).toBe('label-h1-fpn');
   expect(decodeResult.formatted.description).toBe('Flight Plan');
-  expect(decodeResult.formatted.items.length).toBe(6);
+  expect(decodeResult.formatted.items.length).toBe(9);
   expect(decodeResult.formatted.items[0].label).toBe('Route Status');
   expect(decodeResult.formatted.items[0].value).toBe('Route Inactive');
   expect(decodeResult.formatted.items[1].label).toBe('Origin');
   expect(decodeResult.formatted.items[1].value).toBe('KEWR');
   expect(decodeResult.formatted.items[2].label).toBe('Destination');
   expect(decodeResult.formatted.items[2].value).toBe('KDFW');
-  expect(decodeResult.formatted.items[3].label).toBe('Aircraft Route');
-  expect(decodeResult.formatted.items[3].value).toBe('VECTOR >> DISCO >> RIVET');
-  expect(decodeResult.formatted.items[4].label).toBe('Aircraft Route');
-  expect(decodeResult.formatted.items[4].value).toBe('TACKE');
-  expect(decodeResult.formatted.items[5].label).toBe('Message Checksum');
-  expect(decodeResult.formatted.items[5].value).toBe('0xc8b5');
+  expect(decodeResult.formatted.items[3].label).toBe('Company Route');
+  expect(decodeResult.formatted.items[3].value).toBe('EWRDFW01(17L): >> SAAME > J6 > HVQ > Q68 > LITTR >> MEEOW >> FEWWW');
```

Comment:
it's a little funky, open to suggestions to changes in `route_utils.ts`

Reply:
I think this is fine for now. Once we see how the formatted versions look on Airframes and AcarsHub and/or get other feedback, we can always refine it. I'll keep you posted.